### DISCUSSION TITLE
Inquiry about Martensitic Variants

### DISCUSSION THREAD

**Discussion by Alphagenius007** [ORTools4MTEX/ORTools#67] [Q&A] [answered]
I have encountered a persistent challenge: my experimental code consistently yields errors, and I have been unable to pinpoint the cause of these issues. This has been a significant obstacle in my research, and it has prompted me to seek guidance from experts like yourself.
The first problem is that a warning will appear when running into the image below, but the image will load.
![image1](https://github.com/ORTools4MTEX/ORTools/assets/95734163/98882154-90e1-440d-8d6f-8c36fe81691b)
The second problem is that some data content will pop up when the grain code is selected in the following line, but the data is not complete, and the column statistics of single grain martensitic variants and their positions in the pole diagram cannot be loaded out.
![image2](https://github.com/ORTools4MTEX/ORTools/assets/95734163/bd37b6e4-fa43-460a-a8a3-8af126dc0728)
How can I fix it

**Comment by frankNiessen** [comment, marked as answer]
Dear [Alphagenius007](https://github.com/Alphagenius007),

in case you are working with martensitic or bainitic steel, you should not be worried about the issued warning. The warning simply expresses that we group the variants into packets following the known relationships in steel, 111 austenite plane parallel to 110 martensite plane. If you work with steel, this is what you want, if you work with another alloy system, say Titanium, you should simply not plot or analyze the packets and you are good. 

For you second issue, it seems that your property `job.csParent.color` is an empty string.
In my case, after having initialized the *parentGrainReconstructor* class (created the *job* object using `job = setParentGrainReconstructor(ebsd,grains,Ini.cifPath);`) I can check the colors and get valid RGB value sets:
```
>> job.csParent.color

ans =

    0.5608    0.7373    0.5608

>> job.csChild.color

ans =

    0.5294    0.8078    0.9804

```

Could you verify this in your case?

Best wishes
Frank

**Reply by Alphagenius007** [reply]
Dear Frank,

I'm writing to update you on the issue we discussed earlier regarding the job.csParent.color property in my project. I'm pleased to inform you that I have successfully resolved the problem. The root cause turned out to be related to the initial selection of the parent (Gamma) and child (AlphaP) phases. Initially, I had independently chosen the (Fe-Gamma) from the ORtools file, which seems to have been the source of the discrepancy.

Your advice was instrumental in guiding me to revisit and scrutinize this part of my setup, leading to the discovery of this oversight. I deeply appreciate your help and the insights you provided, which not only helped in resolving this specific issue but also enhanced my understanding of the nuances involved in working with these materials and tools.

Thank you once again for your valuable support. It's always reassuring to know that there's reliable expertise to rely on when navigating complex challenges in our field.

Best regards,
Chao
